### ISSUE TITLE
How to change font size on legend

### ISSUE THREAD

**Issue by grzanka** [root-project/jsroot#256]
Dear JSROOT developers.

We are trying to employ MultiGraph to combine several TGraphs on a single canvas.

Our code can be found here:
https://github.com/yaptide/ui/pull/1061/files#diff-14ff87ae947ae41fa4cdb544bb4afcf0ba519eca4d43356d3d355a45c4c61333

We've been inspired by following examples:
https://root.cern/js/latest/api.htm#custom_html_tlegend
https://github.com/root-project/jsroot/blob/master/demo/multigraph_legend.htm

In that example and our code as well the font size for the legend is very large, as seen here:
![image](https://github.com/root-project/jsroot/assets/7374733/65492bc8-038f-4a9a-adee-df626c718393)
That is because we haven't specified legend size.

After right click and resetting the legend to default position:
![image](https://github.com/root-project/jsroot/assets/7374733/2bc1f87d-89bb-4ec2-b2a6-21f26af4d497)
it becomes very small:
![image](https://github.com/root-project/jsroot/assets/7374733/1eef6631-dc20-4b52-a640-57ddf1a81dc9)

Later I've realized that the legend size can be controlled via:
```
JSROOT.extend(leg, { fX1NDC: 0.1, fY1NDC:0.5, fX2NDC: 0.5, fY2NDC:0.7 });
```

Is there any option in JSROOT to automatically position legend in an empty space in a plot ?
Then I would need only to specify the legend size.

**Comment by linev** [comment]
Hi,

As I can see, you are using `master` branch or JSROOT.

In master `autoplace` option was implemented, but not accessible via context menu - it was a bug.
I fixed master branch - now context menu item is available.

I also modify `multigraph_legend.htm` demo, which now shows how 'autoplace' draw option can be used.
See here:

https://jsroot.gsi.de/dev/demo/multigraph_legend.htm

Master branch will be released soon (~2 weeks) as version 7.4.0

**Comment by grzanka** [comment]
@linev thanks for advice, we've been able to implement the multigraph following your suggestions.
It works nicely now:
<img width="834" alt="Screenshot 2023-06-10 at 11 26 05" src="https://github.com/root-project/jsroot/assets/7374733/7ba94e5e-cc2d-4c2d-91c9-9959000595e9">
<img width="836" alt="Screenshot 2023-06-10 at 11 26 32" src="https://github.com/root-project/jsroot/assets/7374733/f29c9eb3-2f63-46b6-8181-adfa0b9bb064">

https://github.com/yaptide/ui/pull/1061/files#diff-14ff87ae947ae41fa4cdb544bb4afcf0ba519eca4d43356d3d355a45c4c61333
